topics/jupyter-notebook/index.md

```diff
@@ -1,13 +1,15 @@
 ---
-created_by: Jupyter Developers
+aliases: notebook, jupyter, ipython-notebook, ipynb, ipynb-jupyter-notebook, ipynb-notebook
+created_by: Fernando Pérez, Brian Granger, and Min Ragan-Kelley
 display_name: Jupyter Notebook
 github_url: https://github.com/jupyter/notebook
 logo: jupyter-notebook.png
 released: December 2011
-related: jupyter
+related: ipython, python, machine-learning, data-visualization, kernel
 short_description: Jupyter Notebooks are documents that contain live code, visualizations, and narrative text.
 topic: jupyter-notebook
 url: https://jupyter.org/
+wikipedia_url: https://en.wikipedia.org/wiki/Project_Jupyter#Jupyter_Notebook
 ---
 
-The Jupyter Notebook is a language-agnostic HTML notebook application for Project Jupyter. Jupyter notebooks are documents that allow for creating and sharing live code, equations, visualizations, and narrative text together. People use them for data cleaning and transformation, numerical simulation, statistical modeling, data visualization, machine learning, and much more. 
+The Jupyter Notebook, previously known as the IPython Notebook is a language-agnostic HTML notebook application for Project Jupyter. Jupyter notebooks are documents that allow for creating and sharing live code, equations, visualizations, and narrative text together. People use them for data cleaning and transformation, numerical simulation, statistical modeling, data visualization, machine learning, and much more.
```

Comment:
Tiny typographic suggestion while I'm in here 😇 

```suggestion
The Jupyter Notebook, previously known as the IPython Notebook, is a language-agnostic HTML notebook application for Project Jupyter. Jupyter notebooks are documents that allow for creating and sharing live code, equations, visualizations, and narrative text together. People use them for data cleaning and transformation, numerical simulation, statistical modeling, data visualization, machine learning, and much more.
```

Reply:
I'll apply the change